src/tests/tracing/eventpipe/common/IpcUtils.cs

```diff
@@ -118,11 +117,11 @@ public static async Task<bool> RunSubprocess(Assembly currentAssembly, Dictionar
                     }
                 };
 
-                while (!EventPipeClient.ListAvailablePorts().Contains(process.Id))
-                {
-                    Logger.logger.Log($"Standard Diagnostics Server connection not created yet -> try again in 100 ms");
-                    await Task.Delay(100);
-                }
+                // while (!EventPipeClient.ListAvailablePorts().Contains(process.Id))
```

Comment:
TODO, uncomment and re-add the Microsoft.Diagnostics.Tools.RuntimeClient import

Reply:
This code needs to stay, since CI machines are slow. It can take far longer than on a normal machine for things to take place, including a process to start. They are running thousands of tests and may only have two cores.